pkg/cortex/modules.go

```diff
@@ -75,7 +75,7 @@ const (
 	StoreGateway             string = "store-gateway"
 	MemberlistKV             string = "memberlist-kv"
 	ChunksPurger             string = "chunks-purger"
-	BlocksPurger             string = "blocks-purger"
+	TenantDeletion           string = "blocks-purger"
```

Comment:
Can we also safely rename the value `blocks-purger`?

Reply:
Yes. Given that it's marked as "invisible" to user and thus considered internal, I think that's perfectly fine.